relay-config/src/config.rs

```diff
@@ -277,6 +277,38 @@ impl ConfigObject for Credentials {
     }
 }
 
+/// Information on a downstream Relay.
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct StaticRelayInfo {
+    /// the id of the relay
+    pub id: RelayId,
+
+    /// The public key that this Relay uses to authenticate and sign requests.
+    pub public_key: PublicKey,
+
+    /// Marks an internal relay that has privileged access to more project configuration.
+    #[serde(default)]
+    pub internal: bool,
```

Comment:
At some point we had discussed that these Relays are trusted by definition, similar to how this was handled in Sentry. Did you find new reasons to include this flag?

Reply:
Yes, we did discuss but I see no reason/downside to limit it to trusted relays. 
The reasons I implemented it like that are:
* makes it clear from the configuration how the relays are treated
* makes it possible for us ( or more probable a large enterprise) to chain its own relays without having the downstream relays depend on registration). This would enable a similar scenario ( the org to bring up additional downstream relays during an incident)

So giving the above and the fact that it does not complicate the handling in any way (we create a Relay object and we have to give it an internal flag, rather than hard coding it we take it from config) I thought it would be worth it having it in the configuration,

The only downside I can think of is that we need another entry (the internal flag) in the configuration file. The field has a default ( which is false, for safety) so although it can be omitted in our use case we need overwritten  to `true` .